appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1544,7 +1482,13 @@ Blockly.ReplMgr.getFromRendezvous = function() {
                       return;   // We're done
                     };
 
-                    rs.state = Blockly.ReplMgr.rsState.CONNECTED;
+                    var phoneState = top.ReplState.phoneState;
+                    if (!phoneState.initialized) {
+                        phoneState.initialized = true;
```

Comment:
Maybe we can file this as technical debt, but since this is repeated in a couple places in replmgr we might want to refactor it into a separate reset method (maybe `reinitialize()`, `clearCodeCache()`, or something similar).